package.json

```
@@ -1,23 +1,54 @@
 {
-  "name": "holochain-rad-phase-2",
+  "name": "holochain-rad-tools-phase-2",
+  "version": "0.1.0",
   "description": "The second tier of RAD tools, automating a UI GraphQL and DNA generation based on a JSON schema file.",
-  "version": "0.0.1",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "generate:ui": "node ui/scripts/generate-ui.js",
-    "generate:clean": "rm -rf ui/generated_ui"
+    "start": "npm run start:sim2h && npm run hc:start; npm run ui:start",
+    "start:sim2h": "sim2h_server -p 9000 &> sim2h-server.log &",
+    "happ:install": "cd dna-src/test && npm i && cd ../../ui-src && npm i && cd ..",
+    "happ:generate": "npm run happ:remove && npm run ui:generate && npm run hc:generate-dna",
+    "happ:remove": "rm -rf dna-src/ ui-src/ .cargo/ target/",
+    "ui:start": "cd ui-src && npm run ui:connect-to-conductor",
+    "ui:generate": "node src/ui-setup/scripts/generate-ui",
+    "ui:generate-clean": "rm -rf ui-src/*",
+    "hc:clean-storage": "rm -rf ./.holochain/holo/storage",
+    "hc:generate-dna": "node src/dna-setup/scripts/generate-dna",
+    "hc:config-add-agent": "node setup/conductor-setup/add-agent-to-conductor",
+    "hc:keygen": "node setup/conductor-setup/create-new-agent",
+    "hc:package": "cd dna-src && hc package",
+    "hc:build": "npm run hc:package",
+    "hc:test": "cd dna-src && RUST_BACKTRACE=1 HC_IGNORE_SIM2H_URL_PROPERTY=true hc test | node test/node_modules/faucet/bin/cmd.js",
+    "hc:start": "holochain -c conductor-config.toml &> conductor.log &",
```

It's a little awkward that this hides its output. It means it fails silently when you haven't generated a conductor. We could add a check to this command that returns an error if there's no conductor, saying "try running hc:generate-conductor first".

But actually I think it's better to not send the output to a log file. Let the user do that if they want.